addons/project/res_partner.py

```diff
@@ -36,6 +36,7 @@ def _task_count(self, cr, uid, ids, field_name, arg, context=None):
         'task_count': fields.function(_task_count, string='# Tasks', type='integer'),
     }
 
+    @api.cr_uid_id_context
```

Comment:
I also prefer `record_id` or `rec_id` (another inconsistency across Odoo code...) because `id` is a reserved word.